src/main/java/edu/cornell/kfs/gl/batch/service/impl/CuPostExpenditureTransaction.java

```diff
@@ -0,0 +1,53 @@
+package edu.cornell.kfs.gl.batch.service.impl;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.kuali.kfs.coa.businessobject.Account;
+import org.kuali.kfs.coa.businessobject.IndirectCostRecoveryExclusionAccount;
+import org.kuali.kfs.coa.businessobject.ObjectCode;
+import org.kuali.kfs.gl.batch.service.impl.PostExpenditureTransaction;
+import org.kuali.kfs.sys.KFSPropertyConstants;
+import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.rice.krad.service.BusinessObjectService;
+import org.kuali.rice.krad.util.ObjectUtils;
+
+public class CuPostExpenditureTransaction extends PostExpenditureTransaction {
+	
+	private static org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(CuPostExpenditureTransaction.class);
+	public static final String ACTIVE_INDICATOR_FIELD = "ACTV_IND";
```

Comment:
Any particular reason why the table column name is being referenced here instead of the property name? I would recommend using the existing equivalent Rice or KFS property name constant, unless it only works when the table column is specified.